README.md

```diff
@@ -1,83 +1,165 @@
 # lambda-handlers
 
-[![Version](https://img.shields.io/pypi/v/lambda-handlers.svg)](https://img.shields.io/pypi/v/lambda-handlers.svg)
-[![Build Status](https://travis-ci.org/enter-at/lambda-handlers.svg?branch=master)](https://travis-ci.org/enter-at/lambda-handlers)
-[![Maintainability](https://api.codeclimate.com/v1/badges/a39e55b85bfcc31204b9/maintainability)](https://codeclimate.com/github/enter-at/lambda-handlers/maintainability)
-[![Test Coverage](https://api.codeclimate.com/v1/badges/a39e55b85bfcc31204b9/test_coverage)](https://codeclimate.com/github/enter-at/lambda-handlers/test_coverage)
-[![Requirements Status](https://requires.io/github/enter-at/lambda-handlers/requirements.svg?branch=master)](https://requires.io/github/enter-at/lambda-handlers/requirements/?branch=master)
-[![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
+[![](https://img.shields.io/pypi/v/lambda-handlers.svg)](https://pypi.org/project/lambda-handlers/)
```

Comment:
what is the reasoning in removing the titles?

Reply:
when rendering to HTML the titles appeared next to the icons, this was the only way I found to not have them there. If you want, please check that with:
```
tox -e docs
open .tox/docs/tmp/html/index.html
```

Reply:
this s***s but ok...